### PR TITLE
model, seed, and route changes for Product-Category vertical

### PR DESCRIPTION
**Pull request by dhernandorena**
Diego's work so far on this request:

- model changes for ProductCategory vertical
- seed changes form ProductCategory vertical
- routes for ProductCategory vertical

need to keep on going on the react and redux components of this...

### CODE REVIEW

File: server/api/productCategories.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router()
```
[the remark this answers is not in the document]
Comment (dhernandorena) [reply server/api/productCategories.js]: If it is all the same to everyone else I kind of prefer to keep it separate so I can follow along with my work.

**Comment by pckilgore** [comment]
Diego, can you rebase this with master like we discussed?  I don't know why the tests are failing, that may fix it.

File: server/db/models/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const User = require('./user')
 const Product = require('./product')
 const Category = require('./category')
+const ProductCategory = require('./productCategory')
```
[the remark this answers is not in the document]
Comment (dhernandorena) [reply server/db/models/index.js]: It is in there

File: server/api/productCategories.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pckilgore) [reply server/api/productCategories.js]: :+1:

File: server/db/models/index.js [hunk 3]
```diff
@@ -6,8 +6,8 @@ const Order = require('./order')
 const OrderLineItem = require('./orderLineItem')
 const Review = require('./review')
 
-Product.belongsToMany(Category, {through: 'productCategories'})
-Category.belongsToMany(Product, {through: 'productCategories'})
+Product.belongsToMany(Category, {through: ProductCategory})
+Category.belongsToMany(Product, {through: ProductCategory})
```
Comment (pckilgore) [inline server/db/models/index.js]: :uruguay:

File: server/db/models/productCategory.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
 const Sequelize = require('sequelize')
 
-const ProductCategory  = db.define('productCategories')
+const ProductCategory  = db.define('productCategories', {
+    id: {
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+    }
+})
```
Comment (pckilgore) [inline server/db/models/productCategory.js]: This is cool!

**Comment by pckilgore** [comment]
Closes #82

**Pull request merged**